core/src/main/java/tc/oc/pgm/spawns/SpawnModule.java

```diff
@@ -68,29 +74,46 @@ public SpawnModule parse(MapFactory factory, Logger logger, Document doc)
         throw new InvalidXMLException("map must have a single default spawn", doc);
       }
 
-      return new SpawnModule(parser.getDefaultSpawn(), spawns, parseRespawnOptions(doc));
-    }
-  }
+      List<RespawnOptions> respawnOptions = Lists.newArrayList();
+
+      for (Element elRespawn :
+          XMLUtils.flattenElements(doc.getRootElement(), "respawns", "respawn")) {
+        respawnOptions.add(parseOptions(elRespawn, true, false));
+      }
+
+      respawnOptions.add(
+          parseOptions(
+              doc.getRootElement().getChild("respawn"),
+              false,
+              doc.getRootElement().getChild("autorespawn") == null));
```

Comment:
This is incorrect, there is not a single respawn child, there's N several respawn children that all generate a default respawn option. 

Consider this code the correct one:

https://github.com/OvercastNetwork/ProjectAres/blob/master/PGM/src/main/java/tc/oc/pgm/spawns/SpawnModule.java#L106